ddtrace/profiling/collector/stack.pyx

```diff
@@ -240,22 +240,26 @@ cdef collect_threads(thread_id_ignore_list, thread_time, thread_span_links) with
                     tstate = PyInterpreterState_ThreadHead(interp)
                     while tstate:
                         # The frame can be NULL
-                        # Python 3.9 added helper function to public C API to access PyFrameObject from tstate,
                         # Python 3.11 moved PyFrameObject to internal C API and cannot be directly accessed from tstate
-                        IF PY_MINOR_VERSION >= 9:
+                        IF PY_MINOR_VERSION >= 11:
```

Comment:
Do we need a else if here for `PY_MINOR_VERSION >= 9` ?

Reply:
This *should* be fine, since `PyThreadState_GetFrame()` was released in 3.9 as an alternative to the deprecation of directly accessing the frame from tstate. Otherwise for `Python<9` we should still be able to grab the frame directly from tstate as we have previously.